### PR TITLE
Add liburcu

### PR DESCRIPTION
**Pull request by ZimmerA**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ZimmerA** [comment]
@conda-forge-admin, please ping team

File: recipes/liburcu/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  version: {{ version }}
+
+source:
+  url: https://lttng.org/files/urcu/userspace-rcu-latest-{{ version }}.tar.bz2
+  sha256: bbfaead0345642b97e0de90f889dfbab4b2643a6a5e5c6bb59cd0d26fc0bcd0e
+
+build:
+  number: 0
+  skip: true  # [osx]
+  skip: true  # [win]
```
Comment (ocefpaf) [inline recipes/liburcu/meta.yaml:14]: These two com be in one line. `skip: True  # [osx or win]`

File: recipes/liburcu/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+
+test:
+  commands:
+    - conda inspect linkages -p $PREFIX $PKG_NAME
```
Comment (ocefpaf) [inline recipes/liburcu/meta.yaml:23]: Conda inspect is deprecated. No need to run it here b/c that is run by default now in all builds.

**Comment by ZimmerA** [comment]
Should be good now!

**Pull request merged**